genrl/deep/agents/a2c/a2c.py

```diff
@@ -84,15 +89,17 @@ def __init__(
             rollout_size,
             **kwargs
         )
-
+        self.network_type = network_type
```

Comment:
Maybe rename this to just `self.network`. Doesn't make sense calling a custom class network_type. (would have to be replaced for every instance of `network_type` tho)